docs/tools/pizza-cli/pizza-action.md

```diff
@@ -0,0 +1,52 @@
+---
+id: pizza-action
+title: "Pizza CLI Action"
+sidebar_label: "Pizza Action"
+keywords: 
+- "codeowners" 
+- "codeowners automation" 
+- "CLI codeowners" 
+- "GitHub Actions"
+- "Codeowners GitHub Action"
+---
+
+To automatically update your CODEOWNERS file every week, you can use the OpenSauced Pizza GitHub Action. This action runs the Pizza CLI in a GitHub Actions workflow.
+
+To set up the action:
+
+1. Create a new file in your repository at `.github/workflows/pizza-action.yml`
+2. Add the following content to the file:
+
+```yaml
+name: OpenSauced Pizza Action
+
+on:
+  schedule:
+    # Run once a week on Sunday at 00:00 UTC
+    - cron: "0 0 * * 0"
+  workflow_dispatch: # Allow manual triggering
+
+jobs:
+  pizza-action:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pizza Action
+        uses: open-sauced/pizza-action@v1.0.0
+        with:
+          # optional and false by default
+          commit-and-pr: "true"
+```
+
+This workflow will run the Pizza CLI's "generate codeowners ./" command once a week, ensuring your CODEOWNERS file stays up-to-date.
```

Comment:
Let's use back ticks instead of double quotes for consistency.

```suggestion
This workflow will run the Pizza CLI's `generate codeowners ./` command once a week, ensuring your CODEOWNERS file stays up-to-date.
```